src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -749,3 +749,10 @@ COOP_PINVOKE_HELPER(void, RhGetCurrentThreadStackBounds, (PTR_VOID * ppStackLow,
 {
     ThreadStore::GetCurrentThread()->GetStackBounds(ppStackLow, ppStackHigh);
 }
+
+ThreadExitCallback g_threadExitCallback;
```

Comment:
I would not do any kind of unification until we have a solid and well tested implementation on Unix.